gemfirexd/core/src/main/java/com/pivotal/gemfirexd/internal/impl/jdbc/EmbedConnection.java

```diff
@@ -1246,6 +1251,12 @@ private void checkUserCredentials(String dbname,
 									  Properties userInfo)
 	  throws SQLException
 	{
+		String user = userInfo.getProperty(com.pivotal.gemfirexd.Attribute.USERNAME_ATTR, "");
+		Misc.getI18NLogWriter().info(LocalizedStrings.DEBUG, "ABS checkUserCredentials(), dbname " +
+				dbname + ", " + user);
+		if ("snappy_hive_metastore".equalsIgnoreCase(user) && "snappydata".equalsIgnoreCase(dbname)) {
+			return;
+		}
```

Comment:
This is the wrong way to bypass credentials and a potential security hole.

Create connection with AuthenticationServiceBase.getPeerAuthenticationService().getBootCredentials() like in GfxdConnectionWrapper.createConnection